CONTRIBUTING.md

```diff
@@ -160,12 +84,13 @@ ready for merge.
 
 ### Merge
 
-Project maintainers merge pull requests using one of the following options:
+Project maintainers merge pull requests using [Squash and merge] and using PR
+description as the commit message, replacing all individual comments made per
+commit.  Authors of the change must ensure PR description is up to date at the
+merge stage, as sometimes comments addressed during code reviews can invalidate
+original PR description.
 
-- [Rebase and merge] The preferable choice for PRs containing a single commit
-- [Squash and merge] Used when there are multiple commits in the PR
-  - Squashing is done to make sure that the project is buildable on any commit
-- [Create a merge commit] Used for LLVM pull-down PRs to preserve hashes of the
-commits pulled from the LLVM community repository
+Pulldown from LLVM upstream is done through merge commits to preserve hashes of
+the original commits pulled from the LLVM community repository.
 
```

Comment:
I think it's worth mentioning @intel/llvm-gatekeepers team can be called out to help with PR merging.